tests/php/test-helpers.php

```diff
@@ -38,4 +38,57 @@ public function test_block_field() {
 		$this->assertEquals( $mock_login, $actual_user_login );
 		$this->assertEquals( $return_value, $actual_user_login);
 	}
+
+	/**
+	 * Test block_lab_enqueue_styles
+	 *
+	 * @covers ::block_lab_enqueue_styles()
+	 */
+	public function test_block_lab_enqueue_styles() {
+		global $wp_styles;
```

Comment:
Sometimes I'll use `$wp_styles = wp_styles();`, but the effect is the same.

Reply:
Updated.